### PR TITLE
Refactoring of components

### PR DESCRIPTION
**Pull request by kvoncode**
Please merge this last, after https://github.com/DarthVanger/NeuralNotes/pull/169 and https://github.com/DarthVanger/NeuralNotes/pull/170

### CODE REVIEW

**Comment by DarthVanger** [comment]
Do you mean RefactOR? :)

**Comment by kvoncode** [comment]
> Do you mean RefactOR? :)

Yes. Refactoring of components

**Comment by DarthVanger** [comment]
> > Do you mean RefactOR? :)
> 
> Yes. Refactoring of components

I mean should the PR title be "Refactor components" instead of "Refact components" ?

**Comment by kvoncode** [comment]
> > > Do you mean RefactOR? :)
> > 
> > 
> > Yes. Refactoring of components
> 
> I mean should the PR title be "Refactor components" instead of "Refact components" ?

Ok. changed

**Comment by kvoncode** [comment]
Please merge this last, after https://github.com/DarthVanger/NeuralNotes/pull/169 and https://github.com/DarthVanger/NeuralNotes/pull/170

File: src/components/LoginPage/LoginPageSlice.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  name: 'login',
+  initialState: {
+    gapiInitialized: false,
+  },
+  reducers: {
+    initGapi: state => {
+      state.gapiInitialized = true;
+    },
+    requestAuth: state => state,
+    authSuccess: state => state,
```
Comment (DarthVanger) [inline src/components/LoginPage/LoginPageSlice.js:13]: What is the point in having a reducer which does nothing?

**Comment by kvoncode** [comment]
Will try

**Pull request closed without merging**